gpytorch/mlls/variational_elbo.py

```diff
@@ -23,7 +24,7 @@ def __init__(self, likelihood, model, num_data, combine_terms=True):
         self.num_data = num_data
 
     def forward(self, variational_dist_f, target, **kwargs):
-        num_batch = target.size(0)
+        num_batch = variational_dist_f.event_shape[0]

```

Comment:
@gpleiss is this really correct?

Reply:
hmm in my code, which has multiple GPs batched together, it doesn't seem to return the size of the mini-batch. seems to always be equal to 1. although maybe there's some other issue on my end...

Reply:
Oh I think I found the issue - I think we have a mean that's not squeezed properly

Reply:
Just pushed to master, should be fixed.

Reply:
ah thanks. that introduced other errors for me but maybe that's on me...

Reply:
@gpleiss update: fixed my problem, thanks!

Reply:
Awesome!